src/controllers/clients.js

```diff
@@ -48,4 +49,21 @@ const getAllClients = async (req, res) => {
 	res.json({ clients });
 };
 
-module.exports = { createClient, getClientByID, updateClient, deleteClient, getAllClients };
+const deleteAllVisitsByClientId = async (req, res) => {
+	const { clientID } = req.params;
+
+	const visits = deleteAllVisitsByClientIdService(clientID);
+
+	visits

```

Comment:
I don't think it'll work, because I suppose method .deleteMany returns an empty array or array of 1+ deleted documents.
So, if you'll get an empty array or object - it'll be true. Our conditional operation never will work in two ways.

Reply:
Yes, thats right. What do you propose to return in response instead of this check?

Reply:
If u need two ways of this endpoint - u can check an empty or not, etc.
It's ur engineering image)

Reply:
now the response is status 204 no content